### ISSUE TITLE
Write tags and versions of lsst_sims, throughputs, and sims_skybrightness_data to imSim FITS headers

### ISSUE THREAD

**Issue by jchiang87** [LSSTDESC/imSim#231]
Some example code on how to do this with the `eups` module in python:
```
In [11]: import eups
                                                            
In [12]: eupsenv = eups.Eups()
                                                  
In [13]: lsst_sims = eupsenv.findProducts('lsst_sims')[0] 
                      
In [14]: lsst_sims.version                                                      
Out[14]: '2.13.0.sims-3-gec5618e+27'

In [15]: lsst_sims.tags                                                         
Out[15]: ['current', 'sims_w_2019_37']

In [16]: throughputs = eupsenv.findProducts('throughputs')                      

In [17]: throughputs                                                            
Out[17]: [Product: throughputs 1.4, Product: throughputs 2019.07.02-1-g0b2113e]

In [18]: for item in throughputs: 
    ...:     print(item.tags) 
    ...:                                                                        
['DC2production', 'current']
['sims_w_2019_37', 'sims_w_2019_34', 'sims_w_2019_36', 'sims_w_2019_35']
```

**Comment by cwwalter** [comment]
So, we are OK with this solution (writing version and tags for lsst_sims and the two throughputs)?

If so, we just need to arrange to have the two throughput related directories tagged with numeric thought tags at the right points.

**Comment by jchiang87** [comment]
For non-metapackages (e.g., `sims_skybrightness_data` and `throughputs`), I think we actually just need the "version" reported by eups since those do indeed seem to correspond to the desired git commits.  For `lsst_sims`, the eups version isn't useful, but it's not clear to me that the eups tag is sufficient either.

**Comment by jchiang87** [comment]
closed by #232
